libs/atqr/domain/src/lib/domain-spec/stories/judge/selectNewJudge.disl

```diff
@@ -0,0 +1,65 @@
+💬 SelectNewJudge! challenge: Challenge
+    NotificationService.SelectNewJudge(challenge)
+
+💬 SupervisionRequest! challenge: Challenge, contactJudge: Contact
+    NotificationService.SupervisionRequested(challenge, contactJudge)
+    
+
+
+service NotificationService
+
+    💬 SelectNewJudge! challenge: Challenge
+        sendEmail
+          template: SelectNewJudge
+          data: Challenge
+          to: challenge.player
+        when get the response   📣 supervisionRequest
+        if(pass X days send again, if pass Y days cancel)
+
+    💬 SupervisionRequested! challenge: Challenge ,contactJudge: Contact
+        sendEmail
+          template: JudgeRequested
+          data: Challenge
+          to: contactJudge
+        when get the response   📣 judgeAccepted or judgeRejected
+        if(pass X days send again, if pass Y days cancel)
+
+
+entity User
+
+  💬 addOfficiatedChallenge! challenge: Challenge
+      officiation = new Officiation(this, challenge)
+      this.officiatedChallenges.push(officiation)
+
+      📣 addJudge! officiation: Officiation
+
+
+entity Challenge
+
+    💬 changeParticipationStatus? participationStatus: ParticipationStatus , judgeId: UserId
```

Comment:
"change" does not capture intention. Please split change into "accept" and "reject" commands